### ISSUE TITLE
is client_id still a constructor requirement?

### ISSUE THREAD

**Issue by slajax** [Retsly/retsly-js-sdk#62]
I don't think it is, if it isn't, lets remove it and just construct from access token for simplicity.

**Comment by wired8** [comment]
+1 on it not being required any more.

**Comment by Jkhwan** [comment]
@slajax it is a requirement for js-auth and we are instantiating js-auth using the sdk instance. Therefore, i believe that we can remove client_id as a constructor requirement on the sdk, but have to refactor js-auth as well to ask for the client_id in its option parameter

**Comment by slajax** [comment]
@Jkhwan good point. Whomever implements this, please create a dependent issue in auth to handle the refactor. They will need to get pulled in together.
